modules/pel/peltool/README.md

```diff
@@ -0,0 +1,72 @@
+### peltool Usage
+To get more peltool's capabilities, use the help option `peltool.py --help`.
+
+**Note:** For non-BMC environment user must pass the PEL directory to parse PELs
+          e.g. `peltool.py -p <pel_dir> -l`, `peltool.py -p <pel_dir> -n`, `peltool.py -p <pel_dir> -a`
+
+#### List PELs
+- Get list of PELs : `peltool.py -l` or `peltool.py -ls` (By default serviceable PELs are listed)
```

Comment:
nit: no space before the ':'.  I see this in a few other places too.

Reply:
also, I think you can just remove the 'peltool.py -ls' example but leave the remark about serviceable listed by default since there's no reason to use the -s.